lab_1_classify_by_unigrams/main.py

```diff
@@ -85,12 +129,31 @@ def compare_profiles(
     'freq' in arguments, None is returned
     """
 
+    # checking input
+    if not isinstance(unknown_profile, dict) or not isinstance(profile_to_compare, dict):
+        return None
+    if 'name' not in unknown_profile.keys() or 'name' not in profile_to_compare.keys():
```

Comment:
you can use only unkwon_profiel without .keys()

Reply:
done